### PR TITLE
Cosmos spells will no longer star mark your steed

### PR DESCRIPTION
**Pull request by carlarctg**
## About The Pull Request

I was wondering wh yi was stuck in place after casting a spell and then eralized my poor cow got a star mark making it unable to pass thru my cosmos. This fixes it so u can be a cowboy heretic

## Why It's Good For The Game

Presumably heretic on horse is cooler than heretic on not-horse

## Changelog

:cl:
fix: Cosmos spells will no longer star mark your steed
/:cl:

### CODE REVIEW

**Comment by Profakos** [comment]
Probably because you committed the image cutter's cache by accident.

File: tools/icon_cutter/cache/hypnagogicv3-1-0.exe
Comment (MrMelbert) [inline tools/icon_cutter/cache/hypnagogicv3-1-0.exe:1]: You committed this

**Review by MrMelbert — CHANGES_REQUESTED** [review]

**Comment by Jacquerel** [comment]
I mean doing this is fine but there's an assload of other spells you could apply this to. Why stop here?

**Comment by carlarctg** [comment]
> I mean doing this is fine but there's an assload of other spells you could apply this to. Why stop here?

ided

also this one 

1. blocks movement which is very important
2. is a 'spell effect' rather than an 'environmental effect' like you might consider a fire spell to indiscriminately attack others

**Pull request merged**